src/components/organisms/AppRouter/AppRouter.tsx

```diff
@@ -77,8 +81,11 @@ const EmergencyViewPage = lazy(() =>
 );
 
 export const AppRouter: React.FC = () => {
+  const { isShown: isModalVisible, hide: hideModal } = useShowHide(!isChrome);
+
   return (
     <Router basename="/">
+      <SwitchToChromeModal show={isModalVisible} onHide={hideModal} />
```

Comment:
Router should have only the routes with the top level component referenced. All the rest components should probably be added inside their respective components